src/main/scala/streamer/TestParameter.scala

```diff
@@ -0,0 +1,18 @@
+package streamer
+
+import chisel3._
+import chisel3.util._
+
+object TemporalAdressGenUnitTestParameters {
+  def temporalLoopDim = 3
+  def temporalLoopBoundWidth = 8
```

Comment:
Shouldn't there be 3 loop bounds for 3 dimensions?

Reply:
Yes. There 8 is the bound width.